justifi-tech/justifi-ruby
# Support creation of business entity

Add module for interacting with [Business Entities](https://docs.justifi.tech/api-spec#tag/Business), due to upcoming deprecation of using `sub_account_id` in the [Hosted Onboarding](https://docs.justifi.tech/api-spec#tag/Hosted-Onboarding) flow. Will resolve issue https://github.com/justifi-tech/justifi-ruby/issues/48.

Module methods mirror the available API methods for the Business entity (create, list, get, update).



Type of Change
--------------

* [ ] Breaking change (would cause existing functionality to not work as expected)
* [ ] This change is in scope for PCI
* [ ] This requires approval from UX
* [ ] This requires approval from Marketing


Steps for Testing/QA
--------------------
Test the following methods on the `Jusifi::Business` module and ensure successful HTTP response:
* `Justifi::Business.create(params: create_params)`
* `Justifi::Business.get(business_id: business_id)`
* `Justifi::Business.list`
* `Justifi::Business.update(business_id: business_id, params: update_params)`
